dagobah/core/core.py

```diff
@@ -1100,3 +1164,23 @@ def _serialize(self, include_run_logs=False, strict_json=False):
         if strict_json:
             result = json.loads(json.dumps(result, cls=StrictJSONEncoder))
         return result
+
+
+class JobTask(object):
+    """ Expandable Task that references a job """
+    def __init__(self, parent_job, job_name, task_name):
+        self.parent_job = parent_job
+        self.job_name = job_name
+        self.name = task_name
```

Comment:
Is there any way for this object to traverse up its parents to determine whether you've given it an actual job name at creation time? It's not validating that anywhere right now, is it?